conf/fluo-deploy.props.example

```diff
@@ -0,0 +1,66 @@
+# Copyright 2014 Fluo authors (see AUTHORS)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+# Cluster user name (install command will SSH to cluster using this user)
+# Leave default below if launching cluster in AWS
+cluster.username = ec2-user
+# Cluster base directory where install/ & tarballs/ directories are created
+# Leave default below if launching cluster in AWS
+cluster.base.dir = /home/ec2-user
+# Hostname of node that must direct installation of cluster.  Will be given Public IP if launching in EC2.
+# If not launching in EC2, node must have Public IP that can be reached from your machine.
+# Hostname can be chosen from "nodes" section below.
+leader.hostname = leader
+# Specifies port to use for dynamic forwarding when connecting to leader
+#leader.forwarding.port = 38585
```

Comment:
need more info to indicate that this is for setting up sock proxy when running `bin/fluo-deploy ssh <cluster>`